src/mixins/filing-template-mixin.ts

```diff
@@ -602,6 +641,8 @@ export default class FilingTemplateMixin extends Mixins(DateMixin) {
     // restore Dissolution data
     this.setBusinessAddress(draftFiling.dissolution.custodialOffice)
     this.setDissolutionType(draftFiling.dissolution.dissolutionType)
+    // setting completing party
+    this.setCompletingParty(this.completingParties())
```

Comment:
Would it be simpler to set the completing party in App.vue::fetchData() ([same as in Edit UI](https://github.com/bcgov/business-edit-ui/blob/ac0c0cf8d9caf706dcbeaa8df12c268ca2424e1b/src/App.vue#L507))?

Reply:
This has been changed.

(The purpose of this mixin is to build or parse data from store -> filing or vice-versa. App.vue’s job to fetch data from the APIs and load it into the store, so that seems like a better place to do this.)

Resolving.